scripts/swagger/build.sh

```diff
@@ -44,7 +45,8 @@ function build() {
 
     npm install --legacy-peer-deps
 
-    npm audit fix --force
+    # FIXME: added --audit-level=high to circumvent https://github.com/advisories/GHSA-qrmm-w75w-3wpx
```

Comment:
This comment can go away, and perhaps even `--audit-level=high`, but I'm not sure if it hurts to keep it